docs/core/setup/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Install .NET Core on Windows, Linux, and macOS - .NET Core
+description: Learn where and what to install for .NET Core on Windows, Linux, and macOS versions. Discover the dependencies required to develop, deploy, and run .NET Core apps.
+author: thraka
+ms.author: adegeo
+ms.date: 11/06/2019
+---
+
+# Download and install .NET Core
+
+This article provides information on how to download and install .NET Core. There are two parts to .NET Core, the **runtime** which is used to run apps, and the **SDK** which is used to create apps. The SDK includes the runtime.
```

Comment:
This article doesn't really provide the info, it points you to where find the info

Reply:
True. I'm trying to figure out how to increase the content in this area. The .NET Core team wanted a decision page that they can point people to, to navigate them to various install/setup guides. I think this will grow as we add more content. Just not sure of the experience just yet. I suspect I'll be getting more feedback from the product unit once this content ships.